osu.Game/Screens/Play/Player.cs

```diff
@@ -1024,11 +1024,11 @@ public override bool OnExiting(IScreen next)
         /// </summary>
         /// <param name="score">The <see cref="Scoring.Score"/> to import.</param>
         /// <returns>The imported score.</returns>
-        protected virtual async Task ImportScore(Score score)
+        protected virtual Task ImportScore(Score score)
```

Comment:
Arguably the `Task` can be removed here, but left for potential future use cases.